kernel/kernel-impl/src/main/java/org/sakaiproject/messaging/impl/UserMessagingServiceImpl.java

```diff
@@ -187,8 +189,10 @@ public void init() {
                     }
 
                     pushService = new PushService(publicKeyBase64, privateKeyBase64);
-                    String pushSubject = serverConfigurationService.getString("portal.notifications.push.subject", "");
+                    String defaultSubject = serverConfigurationService.getServerUrl();
```

Comment:
Why's this different to line 163? Also, I'd define this default once, nearer the top of the init method.

Reply:
Pulled. I'll do more testing on iOS. Chrome, FF and Safari on Mac are singing.